### ISSUE TITLE
Overly aggressive /tmp rm on hsperfdata*

### ISSUE THREAD

**Issue by petrowsky** [omega8cc/boa#567]
When running any other java process on a box, rm'ing hsperfdata* is killing the process.

While I don't see it here on github at
https://github.com/omega8cc/boa/blob/master/aegir/tools/system/graceful.sh#L31

My copy of graceful.sh (which may have been pulled down by skynet) is showing the following.

      rm -f -r /tmp/{google*,Google*,drush*,mapshape*,mc-*,hsperfdata*,tmp*,.sass-cache,pear,jetty*}

When looking at /tmp on my box I see the following...

	drwxrwxrwt 2 root       root       4.0K Jan 12 01:33 .ICE-unix/
	drwxrwxrwt 2 root       root       4.0K Jan 12 01:33 .X11-unix/
	srwxrwxrwx 1 root       root          0 Jan 12 18:27 .newrelic.sock=
	drwxr-xr-x 2 root       root       4.0K Jan 14 19:12 .webmin/
	drwxr-xr-x 2 confluence confluence 4.0K Jan 15 00:56 hsperfdata_confluence/
	drwxr-xr-x 2 jetty8     jetty8     4.0K Jan 15 00:02 hsperfdata_jetty8/
	drwxr-xr-x 4 jetty8     jetty8     4.0K Jan 15 00:02 jetty-0.0.0.0-8088-solr.war-_solr-any-/
	drwxr-xr-x 4 jetty8     jetty8     4.0K Jan 15 00:02 jetty-0.0.0.0-8088-spdy.war-_spdy-any-/
	drwxr-xr-x 4 jetty8     jetty8     4.0K Jan 15 00:02 jetty-0.0.0.0-8088-test.war-_-any-/
	-rw-r--r-- 1 root       root        100 Jan 12 18:25 nrinstall-17929.phpi
	-rw-r--r-- 1 root       root        50K Jan 12 18:25 nrinstall-20150112-182543-17929.tar
	-rw-r--r-- 1 root       root        70K Jan 12 18:26 nrinstall-20150112-182601-18502.tar

Uh oh, that rm is dropping my confluence install. :(

May need to get hsperfdata_jetty* specific.

**Comment by petrowsky** [comment]
My bad. It's not in master because it's here on 2.3.8
https://github.com/omega8cc/boa/blob/BOA-2.3.8/aegir/tools/system/graceful.sh#L29

**Comment by omega8cc** [comment]
This has been fixed in HEAD and 2.4.x-dev.

Currently the skynet agent updates hourly if needed only:

```
/bin/websh
/var/xdrago/runner.sh
/var/xdrago/minute.sh
/var/xdrago/clear.sh
```

Plus of course wrappers in /opt/local/bin/*

**Comment by petrowsky** [comment]
Yep. Just found the fix in your massive commit here.
https://github.com/omega8cc/boa/commit/d39f440dadf7bb64c595609de50d176409427dad#diff-dc1409ca7eb81453da0ce2c1189e1f52L28
